.identity/00_main.tf

```diff
@@ -0,0 +1,32 @@
+terraform {
+  required_version = ">=1.3.0"
```

Comment:
1.6.0

Reply:
done in [42a1bd0](https://github.com/pagopa/io-web-profile-backend/pull/32/commits/42a1bd07a2f1b3618634d703ecbf89c979516510)